roles/core/nfs_client/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+nfs_client_role_version: 1.0.4
+
+nfs_client_mount_exported_dirs: true # true=mounted, false=present
```

Comment:
While I understand the role previously enforced `mounted`, I would be extra careful with such boolean variables. Any future RFE to add support for a third value will lead to a breaking change. We had the same issue with ep_access_controle for 1.3.
I still prefer to allow explicit mounted/present/whatever supported state.
@bluebanquise/bluebanquise: I would like comments from other reviewers.

Reply:
I just allowed all possible mount values in state parameter, commit: 4dd6a48

Reply:
👍 +1, all values should be available.

Reply:
So maybe rename it to: `nfs_client_exported_directories_state `? Since this refer to `state `in the module.

Reply:
Variable renamed as requested, commit : 4dd4ed5